### PR TITLE
Add Flux icon

### LINKED ISSUE #7481: Request: Flux CD
### Brand Name

Flux CD

### Website

https://fluxcd.io/

### Popularity Metric

The project currently has 6.9k Github stars.

Here's the project [link](https://github.com/fluxcd/flux). 

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### Contributing

Yes

### LINKED ISSUE #9333: Add Flux Icon
### Brand Name

Flux

### Website

https://fluxcd.io/

### Popularity Metric

The Similarweb rank is 655,025 and the number of GitHub stars of https://github.com/fluxcd/flux2 is ~5,200.

Flux is one of the two most popular GitOps continuous delivery tools with a well-known reputation in the cloud-native space. It is a [graduated CNCF project](https://www.cncf.io/projects/flux/).

### Official Resources for Icon and Color

svg b/w: https://github.com/cncf/artwork/blob/master/projects/flux/icon/black/flux-icon-black.svg
svg color: https://github.com/cncf/artwork/blob/master/projects/flux/icon/color/flux-icon-color.svg

### Additional Comments

Trademark usage: https://www.linuxfoundation.org/legal/trademark-usage

### Contributing

Yes

### PR DESCRIPTION
**Pull request by makkes**
flux

**Issue:** closes #7481 #9333 

**Similarweb rank:** 655,025
**GitHub stars:** ~5,200

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/flux.svg
Comment (adamrusted) [inline icons/flux.svg:1]: I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/e4fba648-1eb6-46d5-8744-3c387464f873) between your PR and the source icon when scaled correctly. The longest side of the icon should be 24px exactly to pass our linters.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4377,6 +4377,12 @@
             "source": "https://flutter.dev/brand",
             "guidelines": "https://flutter.dev/brand"
         },
+        {
+            "title": "Flux",
+            "hex": "5468FF",
+            "source": "https://github.com/cncf/artwork/blob/c2e619cdf85e8bac090ceca7c0834c5cfedf9426/projects/flux/icon/black/flux-icon-black.svg",
+            "guidelines": "https://www.cncf.io/brand-guidelines/"
+        },
```
Comment (adamrusted) [inline _data/simple-icons.json:4385]:
```suggestion
            "guidelines": "https://www.cncf.io/brand-guidelines",
            "license": {
                "type": "custom",
                "url": "https://www.linuxfoundation.org/legal/trademark-usage"
            }
        },
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @makkes!
I know you're keen to get this merged in pretty swiftly to match up with the endoflife.date PR - so please see comments below. If you're struggling with our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) feel free to ping me and I'll help where I can.

File: icons/flux.svg
Comment (makkes) [reply icons/flux.svg:1]: Thanks for your quick review, @adamrusted. I really struggled to get the SVG into a shape that the preview app generated the right preview without showing a completely distorted version of the logo. That's why I edited it in Inkscape. I didn't change the paths but I scaled the logo so maybe that led to it being different now. Is there a way I can generate that overlay image that you linked above myself so I can verify it locally while trying to get the logo into shape?

Comment (adamrusted) [reply icons/flux.svg:1]: No worries - give me a bit of time this morning to dig out the comparison I did, and I'll fire the SVG code across!

Comment (makkes) [reply icons/flux.svg:1]: I pushed another version of the logo in the hopes that it fixes the deviation.

Comment (adamrusted) [reply icons/flux.svg:1]: There's still a [slight difference](https://github.com/simple-icons/simple-icons/assets/1830867/4f09f7a8-21bc-400b-8fee-e00fb6db349f) for me - below is my optimized path that passes linters and matches the source icon. Feel free to use - but I'll have to ping another maintainer to review.

```html
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Flux</title><path d="M11.402 23.747c.154.075.306.154.454.238.181.038.37.004.525-.097l.386-.251c-1.242-.831-2.622-1.251-3.998-1.602l2.633 1.712Zm-7.495-5.783a8.088 8.088 0 0 1-.222-.236.696.696 0 0 0 .112 1.075l2.304 1.498c1.019.422 2.085.686 3.134.944 1.636.403 3.2.79 4.554 1.728l.697-.453c-1.541-1.158-3.327-1.602-5.065-2.03-2.039-.503-3.965-.977-5.514-2.526Zm1.414-1.322-.665.432c.023.024.044.049.068.073 1.702 1.702 3.825 2.225 5.877 2.731 1.778.438 3.469.856 4.9 1.982l.682-.444c-1.612-1.357-3.532-1.834-5.395-2.293-2.019-.497-3.926-.969-5.467-2.481Zm7.502 2.084c1.596.412 3.096.904 4.367 2.036l.67-.436c-1.484-1.396-3.266-1.953-5.037-2.403v.803Zm.698-2.337a64.695 64.695 0 0 1-.698-.174v.802l.512.127c2.039.503 3.965.978 5.514 2.526l.007.009.663-.431c-.041-.042-.079-.086-.121-.128-1.702-1.701-3.824-2.225-5.877-2.731Zm-.698-1.928v.816c.624.19 1.255.347 1.879.501 2.039.502 3.965.977 5.513 2.526.077.077.153.157.226.239a.704.704 0 0 0-.238-.911l-3.064-1.992c-.744-.245-1.502-.433-2.251-.618a31.436 31.436 0 0 1-2.065-.561Zm-1.646 3.049c-1.526-.4-2.96-.888-4.185-1.955l-.674.439c1.439 1.326 3.151 1.88 4.859 2.319v-.803Zm0-1.772a8.543 8.543 0 0 1-2.492-1.283l-.686.446c.975.804 2.061 1.293 3.178 1.655v-.818Zm0-1.946a7.59 7.59 0 0 1-.776-.453l-.701.456c.462.337.957.627 1.477.865v-.868Zm3.533.269-1.887-1.226v.581c.614.257 1.244.473 1.887.645Zm5.493-8.863L12.381.112a.705.705 0 0 0-.762 0L3.797 5.198a.698.698 0 0 0 0 1.171l7.38 4.797V7.678a.414.414 0 0 0-.412-.412h-.543a.413.413 0 0 1-.356-.617l1.777-3.079a.412.412 0 0 1 .714 0l1.777 3.079a.413.413 0 0 1-.356.617h-.543a.414.414 0 0 0-.412.412v3.488l7.38-4.797a.7.7 0 0 0 0-1.171Z"/></svg>
```

Comment (makkes) [reply icons/flux.svg:1]: Wow thanks, much appreciated. I pushed a commit with your path. Feel free to haul in another maintainer.

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**